### PR TITLE
Add 2 Domains

### PR DESCRIPTION
**Pull request by ukriu**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [ ] You have completed your website. 
- [ ] The website is reachable.  
- [x] You're not using Vercel or Netlify.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.  

## Website Link/Preview

For Emails. Currently.

### CODE REVIEW

**Review by VaibhavSys — CHANGES_REQUESTED** [review]
door.is-a.dev does not exist

**Comment by ukrioo** [comment]
For your information:
It is a email game, a scavenger hunt type of thing, played via back and forth emails.
And the reason I need to have a Github Pages site linked to the nested domain is to be able to start the game.

File: domains/the.neighbour.next.door.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
+{
+    "description": "A fun meme site :0 (as of entering of this domain, it hasn't been published yet).",
+    "owner": {
+        "username": "theNeighbourNextDoor",
+        "email": "theNeighbourNextDoor@ukrio.is-a.dev",
```
Comment (CuteDog5695) [inline domains/the.neighbour.next.door.json:5]: remove the comma here at email line

File: domains/theneighbournextdoor.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "description": "A fun meme site :0 (as of entering of this domain, it hasn't been published yet).",
+    "owner": {
+        "username": "theNeighbourNextDoor",
+        "email": "theNeighbourNextDoor@ukrio.is-a.dev",
```
Comment (CuteDog5695) [inline domains/theneighbournextdoor.json:5]: remove the comma here at email line

**Review by iostpa — DISMISSED** [review]

**Review by VaibhavSys — APPROVED** [review]

**Pull request merged**